src/Orchard.Cms.Web/Modules/Orchard.Contents/Controllers/AdminController.cs

```diff
@@ -474,6 +475,39 @@ public Task<IActionResult> EditPOST(string contentItemId, string returnUrl)
                 return Unauthorized();
             }
 
+            ContentTypeDefinition typeDefinition = _contentDefinitionManager.GetTypeDefinition(contentItem.ContentType);
```

Comment:
Indeed, it's to choose the version on which data will be updated. But maybe it's quite the same for the `Draftable` option which is checked in `ContentsDriver.Edit()` just to show or not the `Save Draft` button. But, even if a content type is `Not Draftable`, you can still create a new draft version through the API.

I think that's why in another admin controller i've seen the following which needs to check the `Draftable` option.

            if (!contentTypeDefinition.Settings.ToObject<ContentTypeSettings>().Draftable)
            {
                menu = await _contentManager.GetAsync(menuContentItemId, VersionOptions.Latest);
            }
            else
            {
                menu = await _contentManager.GetAsync(menuContentItemId, VersionOptions.DraftRequired);
            }

But if you want, as for the `Draftable` option, we can also check the `Versionable` option in `ContentsDriver.Edit()`, this just to show the same buttons but which hit others controller actions.